portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,98 @@
 <!DOCTYPE html>
+<!--My menu-->
+<nav class="menu-main">
```

Comment:
While you can put this here, it most certainly doesn't belong here. The `nav` element belongs in the `body` element.

As with CSS, automatic validation tools are helpful. [HTML Tidy](https://www.html-tidy.org) is the longest lived of such tools and would validate that your HTML does not contain unfortunate markup.